### ISSUE TITLE
Problem with loading RasterPropMonitor

### ISSUE THREAD

**Issue by DimiBro9** [Mihara/RasterPropMonitor#685]
There is thingy that is a bug and it says that I must put dll. file into plugins and I did, and it just says that I didn't put it at all. Help!

**Comment by HerrCraziDev** [comment]
Can you provide more details, like for example which DLL seems to be missing ?
Have you checked you have correctly installed RPM's dependencies ( 'RasterPropMonitor Core' and 'ModuleManager' ) ?

**Comment by DimiBro9** [comment]
No, I Just onstalled correctly and its says that I must have DLL file in
that folder, but I puted it there and it didn't work.

uto, 17. dec 2019. 16:47 HerrCrazi <notifications@github.com> je napisao/la:
